microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -18,7 +18,7 @@ toc::[]
 You can deploy {microshift-short} clusters to either on-premise, cloud, or disconnected environments.
 
 // Double check OP system versions
-{microshift-short} {product-version} is supported on {op-system-ostree-first} and {op-system-base-full} 9.2.
+{microshift-short} {product-version} is supported on and {op-system-base-full} 9.2 and 9.3.
```

Comment:
Why was the ostree hint remove? We do support RHEL 9.2 and 9.3, both rpm or ostree based.
The sentence is now incomplete with a dangling "and".

Reply:
Accidental. Put it back.